app/assets/javascripts/discourse/app/templates/preferences/profile.hbs

```diff
@@ -43,27 +43,30 @@
 <div class="clearfix"></div>
 
 {{#if siteSettings.allow_profile_backgrounds}}
-  <div class="control-group pref-profile-bg">
-    <label class="control-label">{{i18n "user.change_profile_background.title"}}</label>
-    <div class="controls">
-      {{image-uploader imageUrl=model.profile_background_upload_url
-                       type="profile_background"}}
+  {{#if canUploadProfileHeader}}
+    <div class="control-group pref-profile-bg">
+      <label class="control-label">{{i18n "user.change_profile_background.title"}}</label>
+      <div class="controls">
+        {{image-uploader imageUrl=model.profile_background_upload_url
+                        type="profile_background"}}
```

Comment:
Trivial but the indentation here is off. It should either be one line or line up with something.

Reply:
said changes done